src/views/AddressModal.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>{{ $t("Shipment address") }}</ion-title>
```

Comment:
Check figma https://www.figma.com/file/bVPRRw282CqGKMdbz7dciH/HC-Ionic-design-system?node-id=16082-41657&t=SNOroSxsoCFUuPAo-0
```suggestion
      <ion-title>{{ $t("Shipping address") }}</ion-title>
```